lizmap/modules/view/locales/en_US/dictionnary.UTF-8.properties

```
@@ -1,3 +1,6 @@
+startup.error=An error occurred. Please contact your system administrator.
+startup.goToProject=Go to projects.
```

```suggestion
startup.error=An error occurred while loading this map. Some necessary resource may temporarily be unavailable. Please try again later.
startup.goToProject=Go back to the home page.
```


* We must not think the user always knows the administrator nor wants to contact it. I think we should not add this suggestion here
* IMHO We should definitely remove any mention of "project" n LWC and speak about maps.